dash/backend/.env.sample

```diff
@@ -109,4 +109,4 @@ IS_LOCAL=True
 #######################################################
 #./config/gatekeeper.ts
 #######################################################
-GATEKEEPER_TEMPLATE_DIR=gatekeeper-templates
+GATEKEEPER_TEMPLATE_DIR=dist/gatekeeper-templates
```

Comment:
Is this change just for local dev?

Reply:
Yes, this is .env.sample, which only exists to be copied and tweaked for people to create their own .env